addons/hashes/fnc_hashFilter.sqf

```diff
@@ -0,0 +1,59 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_hashFilter
+
+Description:
+    Iterate through all keys and values in a Hash.
+    If function returns true, the key is removed from the hash.
```

Comment:
Maybe it should remove it when the function returns false instead. Like `select <CODE>` does.